test/test_generated_site.rb

```diff
@@ -49,7 +49,9 @@ class TestGeneratedSite < JekyllUnitTest
     end
 
     should "include a post with a abbreviated dates" do
-      refute_nil -1, @site.posts.index { |post| post.relative_path == "_posts/2017-2-5-i-dont-like-zeroes.md" }
+      refute_nil @site.posts.index { |post|
+        post.relative_path == "_posts/2017-2-5-i-dont-like-zeroes.md"
+      }
```

Comment:
any reason you decided to depart from the convention of using `do..end` for code blocks?
```ruby
      refute_nil @site.posts.index do |post|
        post.relative_path == "_posts/2017-2-5-i-dont-like-zeroes.md"
     end
```